src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -5,8 +5,18 @@ const ProjectItemWrapper = styled.section`
 		align-self: flex-start;
 		margin-left: -5rem;
 		padding: 0rem;
+		@media screen and (max-width: 1600px) {
+			margin-left: -12rem;
+		}
 		@media screen and (max-width: 992px) {
-			margin-left: 0rem;
+			margin-left: -10rem;
+		}
+		@media screen and (max-width: 768px) {
+			margin-left: -8rem;
+
+		}
+		@media screen and (max-width: 550px) {
+			margin-left: 4rem;
```

Comment:
not centered enough its more of rightly inclined 
![image](https://user-images.githubusercontent.com/95926324/186675672-a8d8bc47-96ff-4b00-b3ea-3444a05a021b.png)

Reply:
Okay I will look at this thanks

Reply:
any updates @Himanshu664 ?

Reply:
@legendarykamal please see new commit